lib/util.js

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright (c) 2017-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const BSON = require('bson');
+
+// max MongoDB document size leaving 5% for data structure overhead
```

Comment:
Can we add some of the links from your gist that support the 5% overhead number here?